device-types/Cisco/8201-32FH.yaml

```diff
@@ -0,0 +1,87 @@
+---
+manufacturer: Cisco
+model: 8201-32FH
+slug: 8201-32fh
+part_number: 8201-32FH
+u_height: 1
+is_full_depth: true
+subdevice_role: parent
+comments: '[Cisco 8000 Series Routers Data Sheet](https://www.cisco.com/c/en/us/products/collateral/routers/8000-series-routers/datasheet-c78-742571.html)'
+console-ports:
+  - name: con 0
+    type: rj-45
+power-ports:
+  - name: PM0
+    type: iec-60320-c14
+    maximum_draw: 2000
+  - name: PM1
+    type: iec-60320-c14
+    maximum_draw: 2000
+interfaces:
+  - name: Mg0/RP0/CPU0/0
+    type: 1000base-t
```

Comment:
Is this the actual name?

Reply:
Looks like it should be:  MgmtEth0/RSP0RP0/CPU0/0

There may also be a:

MgmtEth0/RSP0RP0/CPU0/1
MgmtEth0/RSP1RP1/CPU0/0
MgmtEth0/RSP1RP1/CPU0/1